src/languages/vbnet.js

```diff
@@ -28,23 +28,41 @@ export default function(hljs) {
     ]
   };
 
-  /**
-   * Date Literals consist of a date, a time, or both separated by whitespace, surrounded by #.
-   */
+  /** Date Literals consist of a date, a time, or both separated by whitespace, surrounded by # */
   const DATE = {
     className: 'literal',
     variants: [
       {
-        // Date: #M/D/YYYY# or #YYYY-MM-DD#
-        begin: /# *((\d+[-/]){2}\d+) *#/
+        // #YYYY-MM-DD# (ISO-Date)
+        begin: /# *\d{4}(-\d\d){2} *#/
       },
       {
-        // Time: #H:mm[:ss]# (24h) or #h[:mm[:ss]] A# (12h )
-        begin: /# *((\d+(:\d+){0,2} *(AM|PM)?)) *#/
+        // #M/D/YYYY# (US-Date)
+        begin: /# *(\d{1,2}\/){2}\d{4} *#/
```

Comment:
1/1/2000 ?

Reply:
Oh I see, I think it's harder to read this way. :)

Reply:
https://github.com/highlightjs/highlight.js/blob/24872afce1b2d69d806e88e6b8f706114955db5e/test/markup/vbnet/literals.txt#L9